Techome/Techome/Views/AddCaffeineDetail/AddCaffeineDetailView.swift

```diff
@@ -0,0 +1,390 @@
+//
+//  AddCaffeineDetailView.swift
+//  Techome
+//
+//  Created by 한택환 on 2022/06/11.
+//
+
+import SwiftUI
+
+struct AddCaffeineDetailViewLayoutValue {
+    
+    struct Paddings {
+        static let sectionPadding: CGFloat = 42
+        static let navigationBarToTitlePadding: CGFloat = 34
+        static let sectionTitleToComponentPadding: CGFloat = 15
+        static let cardVerticalPadding: CGFloat = 15
+        static let titleToBrandPadding: CGFloat = 5
+        static let drinkSizeButtonHorizontalPadding: CGFloat = 3
+        static let drinkSizeButtonInsideHorizontalPadding: CGFloat = 44
+        static let drinkSizeButtonInsideVerticalPadding: CGFloat = 13.5
+        static let effectCardVerticalPadding: CGFloat = 15
+        static let effectCardInsideHorizontalPadding: CGFloat = 18
+        static let effectCardInsideVerticalPadding: CGFloat = 14
+        static let effectDividerHorizontalPadding: CGFloat = 10
+        static let effectSectionIconToTextPadding: CGFloat = 6
```

Comment:
```suggestion
        enum EffectCard {
            static let verticalPadding: CGFloat = 15
            static let insideHorizontalPadding: CGFloat = 18
            static let cardInsideVerticalPadding: CGFloat = 14
            static let dividerHorizontalPadding: CGFloat = 10
            static let sectionIconToTextPadding: CGFloat = 6
        }
```

```swift
EffectCard.sectionIconToTextPadding
```
Enum으로 구분되면, 카테고라이징이 되어서 코드를 사용하고, 작성하기에 보기 쉬울 것 같아요 : )